pom.xml

```diff
@@ -145,40 +143,48 @@
 					<groupId>com.google.guava</groupId>
```

Comment:
Please remove 'springfox-swagger2' artifact and I think you can remove also guava, since it was required for it. Just remove guava too, and check if the project still compiles and starts.